tests/files/packet_rockylinux8-calico.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Instance settings
+cloud_image: rockylinux-8
```

Comment:
@oomichi did you push a image for rocky-8 ? AFAIK these are not built automatically.

/cc @floryut

Reply:
ah, I didn't know that.
https://github.com/kubernetes-sigs/kubespray/blob/f82ed24c030c09e819e1ab758f162f6d48763159/tests/cloud_playbooks/roles/packet-ci/templates/vm.yml.j2#L49
shows we need to register a vm image as `quay.io/kubespray/vm-{{ cloud_image }}` and I don't have a permission to push any images into `quay.io/kubespray` at this time.

Reply:
exactly, at this time this is a limitation and we need @floryut to build and push that image before we can use it in any job.

Reply:
I pushed it last time, is there any issue with the image ? 
https://quay.io/repository/kubespray/vm-rockylinux-8

Reply:
Thank you for pushing the image @floryut 
Let me try again.